tests/test_pets.py

```diff
@@ -0,0 +1,200 @@
+import pytest
+
+from src.data.pet.category import Category
+from src.data.pet.pet import Pet
+from src.data.pet.tag import Tag
+
+
+@pytest.mark.get_pet
+def test_finding_of_existing_pet(create_pet_api):
+    api = create_pet_api
+    pet_id = 174232
+    pet = Pet(id=pet_id, category=Category(3, 'Dog'), name='Jiber', photo_urls=[
+            'https://photo1.jpg', 'https://photo2.jpg'], tags=[Tag(1, 'tag1'), Tag(2, 'tag2')], status='availabe')
+    api.delete_pet(pet_id)  # create new pet before getting
+    api.add_pet(pet)
+    response = api.get_pet(pet_id)
+    assert response.status_code == 200, 'Wrong response status code'
+    response_body = response.json()
+    assert api.validate_pet_schema(response_body)
+    assert response_body.get('id') == pet_id, 'Wrong pet ID in response'
+
+
+@pytest.mark.get_pet
+def test_finding_of_unexisting_pet(create_pet_api):
+    api = create_pet_api
+    pet_id = 123817213827318
+    api.delete_pet(pet_id)  # delete pet before getting
+    response = api.get_pet(pet_id)
+    assert response.status_code == 404, 'Wrong response status code'
+    assert response.json() == {
+        "code": 1,
+        "type": "error",
+        "message": "Pet not found"
+    }, 'Wrong error in response'
```

Comment:
I would prefer to store such responses in a separate file. So in case of changes you will update them only there